### PR TITLE
HPCC-27534 enable building with 64bit Clienttools

### PR DESCRIPTION
**Pull request by xwang2713**
Force Clienttools package to x86_64

Signed off by [xiaoming.wang@lexisnexisrisk.com](mailto:xiaoming.wang@lexisnexisrisk.com)

 @GordonSmith please review

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -164,7 +164,7 @@ IF(${ARCH64BIT} EQUAL 1)
 ELSE(${ARCH64BIT} EQUAL 1)
     SET(CPACK_RPM_PACKAGE_ARCHITECTURE "i386")
 ENDIF (${ARCH64BIT} EQUAL 1)
-SET(CPACK_SYSTEM_NAME "${CMAKE_SYSTEM_NAME}-${CPACK_RPM_PACKAGE_ARCHITECTURE}")
+SET(CPACK_SYSTEM_NAME "${CMAKE_SYSTEM_NAME}-x86_64")
```
Comment (GordonSmith) [inline CMakeLists.txt:167]: This will mislabel the ECL IDE installation package.
I think you want to make this change on line 174?

**Review by GordonSmith — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (xwang2713) [reply CMakeLists.txt]: Fixed

**Comment by dehilsterlexis** [comment]
@xwang2713 I can take a look at this this afternoon. Gordon is on vacation until next week.

**Comment by stuartort** [comment]
Does this impact our ability to tag and build Gold today?
________________________________

**Comment by dehilsterlexis** [comment]
@xwang2713 looks good to me

**Pull request merged**